### DISCUSSION TITLE
Copying navigation from one locale to another links items instead of copying them

### DISCUSSION THREAD

**Discussion by maximmax42** [requarks/wiki#5426] [Error / Bug Report]
After copying navigation menu from one locale to another, trying to edit anything in any item in the new locale edits the item in the base locale and vice versa.

Wiki.js ver 2.5.285 in a Docker container on Linux.

https://user-images.githubusercontent.com/2225711/175776487-5062bf47-1dd5-4311-bac3-9ac6987517b1.mp4

**Comment by c2hat** [comment]
As requested in discord, I wanted to open up a new discussion about my problem, then saw this.
I have the same problem.

**Reply by CMEVL** [reply]
I have the same problem too.
Version 2.5.295

**Reply by christianstenico** [reply]
Same problem here too on 2.5.298.

**Reply by twkrol** [reply]
Same problem here. Version 2.5.300
so annoying

**Reply by Polda18** [reply]
Same issue, version 2.5.301. We have our wiki in Czech and English locales, and this makes the navigation useless. Please fix ASAP.

System configuration:
![sysconfig](https://czghost.is-a-virg.in/6xhjeQkl4.png)

**Reply by Polda18** [reply]
Figured that not the menu itself, but each menu item is linked. Found out when I added some new menu items in one locale, didn't show up in the other one. So I was able to fix the linked menu items by deleting the duplicates and creating new items.
